File: app/src/main/java/com/egd/userinterface/controllers/models/ILEDController.java
package com.egd.userinterface.controllers.models;

/**
 * Abstraction for the controller responsible for managing the LEDs.
 *
 * @author Petar Krastev
 * @since 12.11.2017
 */
public interface ILEDController extends IController {

    /**
     * Turns on the LEDs.
     */
    void LEDsON();

    /**
     * Turns off the LEDs.
     */
    void LEDsOFF();
}

File: app/src/main/java/com/egd/userinterface/iot/controllers/interfaces/ILEDController.kt
package com.egd.userinterface.iot.controllers.interfaces

/**
 * Abstraction for the controller responsible for managing the LEDs.
 *
 * @author Petar Krastev
 * @since 12.11.2017
 */
interface ILEDController : IController {

    /**
     * Turns on the LEDs.
     */
    fun LEDsON()

    /**
     * Turns off the LEDs.
     */
    fun LEDsOFF()
}
